entity/templates/src/main/java/package/web/rest/_EntityResource.java

```diff
@@ -40,15 +41,10 @@
 public class <%= entityClass %>Resource {
 
     private final Logger log = LoggerFactory.getLogger(<%= entityClass %>Resource.class);
-
-    @Inject
-    private <%= entityClass %>Repository <%= entityInstance %>Repository;<% if (dto == 'mapstruct') { %>
-
-    @Inject
-    private <%= entityClass %>Mapper <%= entityInstance %>Mapper;<% } %><% if (searchEngine == 'elasticsearch') { %>
-
-    @Inject
-    private <%= entityClass %>SearchRepository <%= entityInstance %>SearchRepository;<% } %>
+    <% var viaService = service != 'no'; -%>
+    <%- include('../../common/inject_template', {viaService: viaService}); -%><%
+    var instanceType = (dto == 'mapstruct') ? entityClass + 'DTO' : entityClass;
+    var instanceName = (dto == 'mapstruct') ? entityInstance + 'DTO' : entityInstance; -%>
 
```

Comment:
No idea why they felt, I just ran them and got all them as errors -> I'll check if you can't reproduce